pkg/console/controllers/clidownloads/controller.go

```diff
@@ -150,16 +150,19 @@ func GetPlatformURL(baseURL, platform, archiveType string) string {
 	return fmt.Sprintf("%s/%s/%s", baseURL, platform, archiveType)
 }
 
-func PlatformBasedOCConsoleCLIDownloads(host, arch, cliDownloadsName string) *v1.ConsoleCLIDownload {
-	baseURL := fmt.Sprintf("%s/%s", util.HTTPS(host), arch)
+func PlatformBasedOCConsoleCLIDownloads(host, cliDownloadsName string) *v1.ConsoleCLIDownload {
```

Comment:
This is good, but will break the e2e now:

https://github.com/openshift/console-operator/blob/master/test/e2e/downloads_test.go#L34

Just need to update that to not pass the `arch` arg.